packages/explat-client/src/types.ts

```diff
@@ -0,0 +1,35 @@
+// ## The data structures
+
+export interface ExperimentAssignment {
+	/**
+	 * The name of the experiment assignment
+	 */
+	experimentName: string;
+	/**
+	 * The name of the assigned variation,
+	 */
+	variationName: string | null;
+	/**
+	 * The timestamp of when this assignment was retrieved.
+	 */
+	retrievedTimestamp: number;
+	/**
+	 * Time to live from when it was retrieved, seconds
+	 */
+	ttl: number;
+}
+
+// ## Abstracting the outside world
+
+export interface Config {
+	fetchExperimentAssignment: ( {
+		experimentName,
+		anonId,
+	}: {
+		experimentName: string;
+		anonId?: string;
```

Comment:
https://github.com/Automattic/wp-calypso/pull/48475#discussion_r565857355 should actually be here: If we change this to `anonId: string | null`, we can simply pass in the return value of `getAnonId()`.